FordLabs/retroquest
# Countdown timer

## Overview
This PR will add the front end code for a countdown-timer as discussed in #349 

Connects #349 

### Demo
![image](https://user-images.githubusercontent.com/61356329/148084531-60daa7ab-d6ff-4ffe-98ba-44589ff569c5.png)


## Testing Instructions
- Create a board and verify that the timer is visible in the team header
- Default time of 5 minutes should be displayed
- Users can utilize the drop select menu to choose preset times of 10, 15, 30, and 45 minutes
- Once desired time has been selected, press the [green] play button, and the timer should begin counting down
- While the timer is in a running state, the user can click the [yellow] pause button which will halt the timer, but not reset it
- While the timer is paused, the user can once again click the play button and the timer should continue from the time that it left off from
- At any time after the timer has been started, the user can click the [red] clear button, which will cause the drop down menu to once again be displayed for the user to select another time
- When the timer runs down to zero, a dialogue box should appear stating such
- From the dialogue box, the user can then select "Okay!" which will simply close the modal, or they can select "One more minute" which will both reset the timer to one minute and automatically start the countdown without the user having to click play again
